tracing-appender/src/rolling/builder.rs

```diff
@@ -0,0 +1,147 @@
+//! Builder struct for `RollingFileAppender`
+//!
+//! Gives access to setting additional options which are not avaible using standard interface.
+//! Currently it is the only way to enable compression of logs.
+use crate::rolling::{create_writer_file, Inner, RollingFileAppender, Rotation};
+use crate::sync::RwLock;
+use std::path::Path;
+use std::sync::atomic::AtomicUsize;
+use time::OffsetDateTime;
+
+#[cfg(feature = "compression_gzip")]
+use {
+    crate::rolling::compression::CompressionConfig, crate::rolling::compression::CompressionOption,
+};
+
+use crate::writer::WriterChannel;
+
+/// Struct for keeping temporary values of `RollingFileAppender`.
+///
+/// Note that `log_directory` and `log_filename_prefix` are obligatory parameters and should
+/// be passed into the constructor of `RollingFileAppenderBuilder`.
+#[derive(Debug, Clone, Eq, PartialEq)]
```

Comment:
is there any reason for this to implement `Eq` and `PartialEq`? if not, i'd prefer not to derive them if there isn't a reason for them...

Reply:
No reason I guess. Not sure when I added this. Fixed.